coordinator/app/src/main/kotlin/net/consensys/zkevm/coordinator/app/config/CoordinatorConfig.kt

```diff
@@ -237,16 +237,16 @@ data class BlobSubmissionConfig(
   val priorityFeePerGasUpperBound: ULong,
   val priorityFeePerGasLowerBound: ULong,
   val maxBlobsToSubmitPerTick: Int = maxBlobsToReturn,
-  // defaults to 6, not supported atm, preparatory work
-  val targetBlobsToSendPerTransaction: Int = 6,
+  // defaults to 9, not supported atm, preparatory work
```

Comment:
```suggestion
  // defaults to 9, not supported atm, preparatory work
```
To be removed once done?

Reply:
@julien-marchand I would need to check with the others, but I believe there might be some other customisation coming? @Filter94 do you recall?